src/scss/_arrow.scss

```diff
@@ -96,7 +132,12 @@
 	}
 }
 
-@mixin oOverlayRightArrow($color: #74736c) {
+/// Right arrow styling
+///
+/// @param {Color} color (oColorsGetColorFor(o-overlay, border))
+///
+/// @requires {mixin} _oOverlayHorizontalArrow
+@mixin oOverlayRightArrow($color: oColorsGetColorFor(o-overlay, border)) {
```

Comment:
Should `oColorsGetColorFor(o-overlay, border)` be assigned to a global variable setting for the whole module (since it's used in multiple places)?